pkgs/by-name/ru/ruff/package.nix

```
@@ -22,26 +24,49 @@ rustPlatform.buildRustPackage rec {
     hash = "sha256-//ayB5ayYM5FqiSXDDns2tIL+PJ0Osvkp8+MEEL0L+8=";
   };
 
-  cargoLock = {
+  postPatch = ''
+    # Do not rely on path lookup at runtime to find the ruff binary
+    substituteInPlace python/ruff/__main__.py \
+      --replace-fail \
+        'ruff_exe = "ruff" + sysconfig.get_config_var("EXE")' \
+        'return "${placeholder "$out"}/bin/ruff"'
+  '';
```

```suggestion
  # Do not rely on path lookup at runtime to find the ruff binary
  postPatch = ''    
    substituteInPlace python/ruff/__main__.py \
      --replace-fail \
        'ruff_exe = "ruff" + sysconfig.get_config_var("EXE")' \
        'return "${placeholder "$out"}/bin/ruff"'
  '';
```

Please, move the comment out of the string.